implementations/rust/ockam/ockam_api/src/nodes/service/secure_channel.rs

```diff
@@ -246,12 +260,14 @@ impl NodeManagerWorker {
         let route = crate::multiaddr_to_route(&addr)
             .ok_or_else(|| ApiError::generic("Invalid Multiaddr"))?;
 
+        let identity_ctx = identity.map(|id| Some((id, ctx))).unwrap_or(None);
```

Comment:
We shouldn't pair the Context with any variable as it's a pretty broad variable that could be useful in different scenarios. Just pass the two variables separately.

Reply:
At first I put both of them separately, but since `create_secure_channel_impl` is called in multiple scopes (e.g. [here](https://github.com/build-trust/ockam/blob/2de6faaac4bc1ee7c7dc06dc914dd7408be11de8/implementations/rust/ockam/ockam_api/src/nodes/service.rs#L349)) this led to various changes to bring `Context` into scope each time `create_secure_channel_impl` is called. Anyway, I have separated both of them and I will push the code now